.github/actions/install-unreal-linux/action.yml

```diff
@@ -27,5 +27,6 @@ runs:
       shell: bash
       run: |
         mkdir -p /mnt/work/${{ inputs.unreal-program-name }}
-        unzip -q ./ue.zip -d /mnt/work/${{ inputs.unreal-program-name }}
+        # Selectively extract files in order to save disk space
```

Comment:
Looks like a straight forward optimization..

... but if there are unneeded files in the linux ue zip, could just make our own .zip on s3 (like the others) that already have the unnecessary files pruned out. Would save on download bandwidth and the extra unzip complexity.

Reply:
This isn't so much an optimization as a desperate attempt to get it to work at all. The GH Actions Linux runner doesn't have enough disk space to extract the whole thing. Yes, we could optimize the ZIP from here.